html/elements/link.json

```diff
@@ -146,19 +146,23 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": true,
+                "notes": "Chrome does not include stylesheets which have the HTML <code>disabled</code> attribute in <code>document.styleSheets</code>."
```

Comment:
This is wrong: Firefox (and the proposed spec) won't include them. Chrome and WebKit don't include them if they're added to the document with the `disabled` attribute, but they do if the attribute is removed dynamically (that's silly).

Reply:
You're right. I misread the output from the test code I wrote, which is tremendously embarrassing. I'll correct this.

Reply:
I could have sworn that this note essentially says what I had understood the situation to be on Chrome, but apparently I'm wrong there. I seem to recall seeing someone say that Chrome needed a patch to match this change to the spec, but now cannot find that reference. Do you know what that change would be, @emilio?

Reply:
Yeah, the chrome behavior is a bit hilarious and hard to reason about.

It won't be included in `document.styleSheets` if by the time it's added to the document it has the `disabled` attribute. That is to say, the following:

```html
<!doctype html>
<link rel="stylesheet" href="foo" disabled>
```

would yield no stylesheet in document.stylesheets.

However, this:

```html
<!doctype html>
<link rel="stylesheet" href="foo" onload="this.setAttribute('disabled', 'true')">
```

Would yield a stylesheet in `document.styleSheets`. *shrug* :)

Reply:
@emilio - So if I understand you correctly, you're right about Chrome not making sense. Basically, what I gather is that

* If the page doesn't have the `disabled` flag set on `<link>` when it is first loaded, the stylesheet is included in `document.styleSheets`. It stays in that list no matter what you do from then on.
* If the `disabled` flag is set on a `<link>` when the HTML is loaded, the stylesheet is not added to `document.styleSheets`, and nothing you do causes it to be added. Somehow the stylesheet is applied without being listed. Creepy.

Otherwise, I think the behavior is about the same. But this is weird and tricky stuff. :)

Reply:
(1) is right. Re. (2) I would've sworn that removing `disabled` puts the link back in `document.styleSheets`, and from then on it cannot be removed anymore. But I could be misremembering.

But yeah, it doesn't make any sense, thus the issue.

Reply:
My testing seems to show (2) to be true. I am going to leave it as it is now and ask @jpmedley to review.

Reply:
So you got this from manual testing?

Reply:
@jpmedley 	- Yeah, manual testing. If I'm wrong then I'm wrong. It certainly wouldn't be the first time. I'd have to pore over my code to understand why, but at this point I'd rather take your word for it. Let me know.

Reply:
I'm satisfied that your understanding is correct. Change as you see fit.